website/client/src/components/userMenu/profileModal.vue

```diff
@@ -54,8 +61,40 @@ export default {
     },
     onHidden () {
       if (this.$route.path !== window.location.pathname) {
-        this.$router.go(-1);
+        this.$router.push({ path: this.$route.path });
```

Comment:
Hi @aevix , I'm seeing a few problems with this, instead of trying to recreate the previous title , my suggestion would be to store it in a variable before it's set [here](https://github.com/HabitRPG/habitica/blob/316ed391e8c92894bdafe31a41bb8ec6d1e3f418/website/client/src/components/userMenu/profile.vue#L902) and then restore it when `onHidden` is called. I made an example here https://github.com/HabitRPG/habitica/issues/12760#issuecomment-725464599

Reply:
I am not sure if I am not doing something wrong but I have tried this already and it won't change the title. Maybe you could try it and see if it works on your end. Maybe its the browser I am using but I tested it on safari and google chrome. document.title from the app.js is being triggered and working via `common:setTitle` but I think like I mentioned in the issue, vue router has some sort of disconnect between dom being rendered and the actual browser updating. Only way I got it to update was by changing it to push.

Reply:
could you post the commit where you tried that and I'll give it a look? thanks

Reply:
I made a commit sorry it took so long to respond I didn't get a chance to sit down and work on it until now.